material-ui/app/routes/about.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Link } from '@remix-run/react';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+export default function About() {
+  return (
+    <React.Fragment>
```

Comment:
Let's use the `<>` shorthand, just like in our other examples

```suggestion
    <>
```